doc/cephadm/install.rst

```diff
@@ -257,6 +257,42 @@ hosts to the cluster. No further steps are necessary.
     # ceph orch daemon add mon newhost1:10.1.2.123
     # ceph orch daemon add mon newhost2:10.1.2.0/24
 
+  .. note::
+     The **apply** command can be confusing. For this reason, we recommend using
+     YAML specifications. 
+
+     Each 'ceph orch apply mon' command supersedes the one before it, which means
+     that if you do not use the proper comma-separated list-based syntax when
+     you want to apply monitors to more than one host, you will clobber your
```

Comment:
I'm not sure "clobber" is the best word for a global audience (unless it's a technical term in this context?), I admittedly had to re-read twice as a native speaker to ensure I understood it properly - especially considering "to clobber" means "to hit". or "to strike" Suggestions: mess up, create a problem, screw-up, make difficult etc.

Reply:
"clobber" in this context means "destructively overwriting": https://en.wikipedia.org/wiki/Clobbering

I'm going to let "clobber" stand. However, you've made me think that maybe this sentence would be better if it were cut up into a few sentences.